src/languages/asciidoc.js

```diff
@@ -9,6 +9,37 @@ Category: markup
+  const P_Nl = unpack_hex_range('16EE-16F02160-21822185-218830073021-30293038-303AA6E6-A6EF')
+  const P_Nd = '0-9' + (unpack_hex_range('0660-066906F0-06F907C0-07C90966-096F09E6-09EF0A66-0A6F0AE6-0AEF0B66-0B6F0BE6-0BEF0C66-0C6F0CE6-0CEF0D66-0D6F0DE6-0DEF0E50-0E590ED0-0ED90F20-0F291040-10491090-109917E0-17E91810-18191946-194F19D0-19D91A80-1A891A90-1A991B50-1B591BB0-1BB91C40-1C491C50-1C59A620-A629A8D0-A8D9A900-A909A9D0-A9D9A9F0-A9F9AA50-AA59ABF0-ABF9FF10-FF19'))
+  const P_Pc = unpack_hex_range('005F203F20402054FE33FE34FE4D-FE4FFF3F')
```

Comment:
Oh wow.  Two steps forward one step back. :-)  This might be a bit much. :-)

Context: https://github.com/highlightjs/highlight.js/issues/2756

We're still up in the air on UTF8 but if we're headed there we should do it properly, not with this kind of hack.

Reply:
You could leave a TODO comment if you wanted.

Reply:
OK, I will use `\w` and remove the associated case.
We are using these unicode ranges in Asciidoctor.js to translate `\p{Word}` (Ruby).

Reply:
Yeah, we have access to the `/u` flag in JS and if we're going to start matching `\p` stuff that's what we should be using, but right now there are some performance concerns.  If you know anything about it and wanted to contribute to that thread help would be welcome. :-)